scripts/requirements.in

```diff
@@ -31,7 +31,7 @@ portpicker
 mobly
 
 # zephyr
-west>=0.9.0
+west>=0.10.1
```

Comment:
Update `requirements.txt` as well?

Reply:
They are already updated. Michael did it some time ago.